src/main/java/org/jabref/logic/importer/fetcher/ComplexSearchQuery.java

```diff
@@ -28,7 +31,7 @@ private ComplexSearchQuery(String defaultField, List<String> authors, List<Strin
         this.singleYear = singleYear;
     }
 
-    public Optional<String> getDefaultField() {
+    public Optional<List<String>> getDefaultFieldPhrases() {
```

Comment:
It's considered bad practice to have an optional around a list. Just return an empty list, instead of an empty optional.